File: src/main/scala/resonantinduction/compatibility/nei/RISmelterRecipeHandler.java
package resonantinduction.compatibility.nei;

import resonant.lib.utility.LanguageUtility;
import resonant.content.factory.resources.RecipeType;

public class RISmelterRecipeHandler extends RITemplateRecipeHandler
{

	@Override
	public String getRecipeName()
	{
		return LanguageUtility.getLocal("resonantinduction.machine.smelter");
	}

	@Override
	public RecipeType getMachine()
	{
		return RecipeType.SMELTER;
	}
}

File: src/main/scala/resonantinduction/compatibility/nei/RISmelterRecipeHandler.scala
package resonantinduction.compatibility.nei

import resonant.lib.utility.LanguageUtility
import resonant.lib.factory.resources.RecipeType

class RISmelterRecipeHandler extends RITemplateRecipeHandler
{
    def getRecipeName: String =
    {
        return LanguageUtility.getLocal("resonantinduction.machine.smelter")
    }

    def getMachine: RecipeType =
    {
        return RecipeType.SMELTER
    }
}